scripts/build_genome_data.sh

```diff
@@ -50,21 +30,23 @@ mkdir -p ${DEST_DIR}
 cd ${DEST_DIR}
 
 if [[ $GENOME == "hg19" ]]; then
+  MITO_CHR_NAME="chrM"
```

Comment:
Thanks for your suggestion, but i would like keep it as is because there is actually no default value for this variable. It's a coincidence that all supported genomes use `chrM` as mito chr name and don't want to introduce an additional var.